CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.2.0] - TBD
```

Comment:
I thought so but then I wasn't sure because its doesn't really include changes that aren't backwards compatible. @patduin @abhimanyugupta07 @JayGreeeen what do you guys think?

Reply:
I am fine with this being a major update. This change is big enough.

Reply:
Let's make it so then ;)